### ISSUE TITLE
Question about dataset

### ISSUE THREAD

**Issue by ecnuycxie** [Norod/U-2-Net-StyleTransfer#4]
Thanks for your great works!

I'd like to ask how to collect paired dataset for ffhq.

**Comment by Norod** [comment]
You generate image pairs using the same seed/vector on the base and fine-tune models. For example, for each seed, generate once on FFHQ and once on your FFHQ fine-tuned model. 

 I once shared an old Tensor-Flow-1.x code which does it, it should be as simple to port it into a modern pytorch version:
https://pastebin.com/xgciynUH

Good luck

**Comment by ecnuycxie** [comment]
This is a interesting and greate way to generate paired data!  Thanks!

> You generate image pairs using the same seed/vector on the base and fine-tune models. For example, for each seed, generate once on FFHQ and once on your FFHQ fine-tuned model.
> 
> I once shared an old Tensor-Flow-1.x code which does it, it should be as simple to port it into a modern pytorch version: https://pastebin.com/xgciynUH
> 
> Good luck

**Comment by ecnuycxie** [comment]
Besides, i'd like to ask how many training samples are used in your training process.

**Comment by Norod** [comment]
It's written in the link I provided in my previous comment 

`number_of_gen_cycles = 1750     #1750 * 4 = 7K image pairs, good for p2s2p, e4e. For Pix2PixHD use a higher number like 6500 (*4 = 26000)`
